en/editor/project/joint-texture-layout.md

```diff
@@ -0,0 +1,71 @@
+# Joint Texture Layout Settings
+
+To ensure that the [Skeletal Animation](../../engine/animation/skeletal-animation.md#About-Dynamic-Instancing) can also participate fully and correctly in [Dynamic Instancing](../../engine/renderable/model-component.md#Instancing-Batching), requiring the user to manually specify how the data for each joint texture is assigned.
```

Comment:
This paragraph doesn't seem to make much sense.

Reply:
@xunyi0 do you have thoughts on this paragraph. I don't quite fully understand what needs to be said here. What do you think should be written?

Reply:
@xunyi0 just bumping this comment for your thoughts

Reply:
@slackmoehrle Maybe this:

To ensure that [skeletal animations]() can fully and correctly involve [dynamic instancing batching](), the developer needs to manually specify the data assignment method for each joint texture.

Reply:
This makes sense! @xunyi0